llvm/test/CodeGen/X86/isel-br.ll

```diff
@@ -0,0 +1,31 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -O0 -mtriple=i686-linux-gnu                                     -verify-machineinstrs | FileCheck %s --check-prefix=DAG
+; RUN: llc < %s -O0 -mtriple=i686-linux-gnu -fast-isel -fast-isel-abort=1       -verify-machineinstrs | FileCheck %s --check-prefix=DAG
+; RUN: llc < %s -O0 -mtriple=i686-linux-gnu -global-isel -global-isel-abort=1   -verify-machineinstrs | FileCheck %s --check-prefix=GISEL
+; RUN: llc < %s -O0 -mtriple=x86_64-linux-gnu                                   -verify-machineinstrs | FileCheck %s --check-prefix=DAG
```

Comment:
use explicit -global-isel=0 for dag cases?

Reply:
Added for SelectionDAG, or you meant FastISel as well?

Reply:
I'm not sure what the flag interaction between -fast-isel and -global-isel is

Reply:
`TargetPassConfig.cpp` checks first checks for FastISel, then for GlobalISel.
https://github.com/llvm/llvm-project/blob/c95693c74601521463e12ff358fed6fb9ee736b9/llvm/lib/CodeGen/TargetPassConfig.cpp#L974
`-global-isel=0` probably makes sense for futureproofing purposes with SelectionDAG... don't have a preference here.